tools/privacyidea-pip-update

```diff
@@ -25,10 +25,8 @@ def update():
 
 def update_db_schema(environment):
     mig_path = environment + "/lib/privacyidea/migrations"
-    # stamp the database
-    call("pi-manage db stamp 4f32a4e1bf33 -d {0}".format(mig_path), shell=True)
     # update the database schema
-    call("pi-manage db upgrade -d {0}".format(mig_path), shell=True)
+    call("privacyidea-schema-upgrade {0}".format(mig_path), shell=True)
```

Comment:
I think we also need to add ``privacyidea-schema-upgrade`` to the scripts section in setup.py.

Reply:
There is no need to do this.
Since I once forgot to add scripts, there is the mechansim that all scriptins in the ``tools/`` directory get included in setup.py.